checks/wmi.py

```diff
@@ -0,0 +1,226 @@
+
```

Comment:
Should we name it `wmi_checks.py` instead to be consistent with `network_checks.py`: I feel like both have a similar role.